docs/_static/vcr/v2.yaml

```diff
@@ -0,0 +1,281 @@
+          content:
+            application/vc+json;type=revocation:
+              schema:
+                $ref: '#/components/schemas/Revocation'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/verifier/vc:
+    post:
+      summary: Verifies a Verifiable Credential
+      description: |
+        Verifies a Verifiable Credential: It checks the signature, expiration, revocation status and if the issuer is trusted.
+        
+        error returns:
+        * 400 - One or more of the given parameters are invalid
+        * 500 - An error occurred while processing the request
+      operationId: "verifyVC"
+      tags:
+        - credential
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/VerifiableCredential'
+      responses:
+        "200":
+          description: "The verification result"
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/VCVerificationResult'
+        default:
+          $ref: '../common/error_response.yaml'
+
+
+components:
+  schemas:
+    IssueVCRequest:
+      type: object
+      description: A request for issuing a new Verifiable Credential.
+      required:
+        - type
+        - issuer
+        - credentialSubject
+      properties:
+        "@context":
+          description: |
+            The resolvable context of the credentialSubject as URI. If omitted, the "https://nuts.nl/credentials/v1" context is used.
+            Note: it is not needed to provide the "https://www.w3.org/2018/credentials/v1" context here.
+          type: string
+          example: "http://schema.org"
+          default: "https://nuts.nl/credentials/v1"
+        type:
+          description: Type definition for the credential.
+          type: string
+          example: "NutsOrganizationCredential"
+        issuer:
+          description: DID according to Nuts specification.
+          type: string
+          example: "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY"
+        expirationDate:
+          description: rfc3339 time string until when the credential is valid.
+          type: string
+          example: "2012-01-02T12:00:00Z"
+        publishToNetwork:
+          description: |
+            If set, the node publishes this credential to the network. This is the default behaviour.
+            When set to false, the caller is responsible for distributing the VC to a holder. When the issuer is
+            also the holder, it then can be used to directly create a presentation (self issued).
+          type: boolean
+          default: true
+        visibility:
+            description: |
+              When publishToNetwork is true, the credential can be published publicly of privately to the holder.
+            type: string
+            enum: [ public, private ]
+            default: private
```

Comment:
devs will forget this for NutsOrganizationCredentials. It's currently taken from the type settings. A different option would be to make this param mandatory.